Nelsonomotuyole.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Nelson omotuyole",
+  "displapstatsPublic": true,
+  "type": "personal",
+  "bio": "Teacher turned **programmer** | Intergrated science student from *The university of lagos Nigeria | presently Learning **phyton pro bootcamp with angela yu**  ",
+  "avatar": "https://github.com/Nelsonomotuyole.png"
+  "Links": [
+    { 
+      "name": "follow me on twitter",
+      "url": "https://twitter.com/nelsonomotuyoIe",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/Nelsonomotuyole",
+      "icon": "FaGitHub"
```

Comment:
this GitHub icon should be `FaGithub`

Reply:
> Sorry Is it 'faGitHub'

nah its `FaGithub` with a small `h`

Reply:
Okay thanks
When it has been merged and approved where do I find the url for my website sir?

Reply:
> Okay thanks When it has been merged and approved where do I find the url for my website sir?

after merge and the release your URL will be `https://linkfree.eddiehub.io/Nelsonomotuyole`

Reply:
its telling me not found on the browser without showing my credentials

Reply:
> i posted the link on a browser but it did not show anything i think it hasnt been approved yet?

yes you need to wait for this to get PR to get merged and released in a new version of Linkfree, releases are pushed every day, so just wait for the merge

Reply:
okay thanks